### PR TITLE
Feat/add UI into dg and fix javadoc

### PR DESCRIPTION
**Pull request by alvynben**
* Adds UI Class Diagrams into DG
* Fixes Javadoc comments which were missing parameters

### CODE REVIEW

File: src/main/java/seedu/fridget/Fridget.java [hunk 1]
```diff
@@ -25,6 +25,9 @@
 
     /**
      * Constructor for Fridget.
```
Comment (zonglun99) [inline src/main/java/seedu/fridget/Fridget.java:27]: can put a white space after description

File: src/main/java/seedu/fridget/Fridget.java [hunk 2]
```diff
@@ -60,6 +63,7 @@ public void run() {
 
     /**
      * Main entry-point for the java.fridget.Fridget application.
+     * @param args Stores any arguments inputted by the user when running Fridget.
```
Comment (zonglun99) [inline src/main/java/seedu/fridget/Fridget.java:66]: Same for here

**Review by zonglun99 — APPROVED** [review]

File: src/main/java/seedu/fridget/Fridget.java (hunk 1 above)
Comment (alvynben) [reply src/main/java/seedu/fridget/Fridget.java:27]: Good catch.

File: src/main/java/seedu/fridget/Fridget.java (hunk 2 above)
Comment (alvynben) [reply src/main/java/seedu/fridget/Fridget.java:68]: Good catch.

File: src/main/java/seedu/ui/Ui.java [hunk 3]
```diff
@@ -140,6 +141,8 @@ public void printReactionToRemovingItem(Item item, int qty) {
      * Suggests the correct item name if input is incomplete.
      *
      * @param predictedItem Only item available containing search term but is not exactly the same.
+     * @return True if the user said 'y' or False if the User said 'n'
```
Comment (zonglun99) [inline src/main/java/seedu/ui/Ui.java:144]: this is missing a full stop might be the cause of checkstyle error

Comment (alvynben) [reply src/main/java/seedu/ui/Ui.java]: Thanks for noticing, wasn't the cause of the error, but I almost forgot to put the period in.

**Pull request merged**